### DISCUSSION TITLE
Should bufls be installable with mason-lspconfig?

### DISCUSSION THREAD

**Discussion by znd4** [mason-org/mason.nvim#574] [Q&A] [answered]
[bufls](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#bufls) is one of the lspconfig servers, but I'm getting `"bufls" is not a valid entry ...`

**Comment by WhoIsSethDaniel** [comment, marked as answer]
you want to use the `mason.nvim` name for this server: `buf-language-server`.

**Reply by znd4** [reply]
Hmm, would it be a bug in `mason-lspconfig` if both give the warning?

```
[mason-lspconfig.nvim] Server "buf-language-server" is not a valid entry in ensure_installed. Make sure to only provide lspconfig server names.
[mason-lspconfig.nvim] Server "bufls" is not a valid entry in ensure_installed. Make sure to only provide lspconfig server names.
```

(sorry, I just realized my original question was super underspecified)

**Reply by williamboman** [reply]
You can find the list of supported servers [here](https://github.com/williamboman/mason-lspconfig.nvim/blob/main/doc/server-mapping.md). It seems like `bufls` was never added in mason-lspconfig, added in https://github.com/williamboman/mason-lspconfig.nvim/pull/85

**Reply by znd4** [reply]
Thanks so much! Sorry for not tracking down that list first
